Libraries/LibJS/Runtime/Value.cpp

```diff
@@ -453,6 +453,19 @@ ThrowCompletionOr<ByteString> Value::to_byte_string(VM& vm) const
     return TRY(to_string(vm)).to_byte_string();
 }
 
+// FIXME: This should return a ByteString
+ThrowCompletionOr<String> Value::to_idl_byte_string(VM& vm) const
+{
+    // This method implements the ByteString conversion specified here: https://webidl.spec.whatwg.org/#ref-for-idl-ByteString%E2%91%A7
+    auto result = TRY(to_string(vm));
+    for (auto character : result.code_points()) {
+        if (character > 0xFF)
+            return vm.throw_completion<TypeError>(ErrorType::InvalidCodePoint);
+    }
+
+    return result;
+}
```

Comment:
This doesn't feel like the best place to put this method. I put it here to minimize the number of IDL generator changes required.

Reply:
Let's put this in LibWeb/WebIDL/AbstractOperations.h

Reply:
I've moved the string conversion functions used by the IDL generator to WebIDL/AbstractOperations.h.